common/install_magma.sh

```diff
@@ -2,6 +2,7 @@
 
 set -eou pipefail
 
+# Check https://anaconda.org/pytorch/magma-cuda${cuda_version_nodot} to see if there's a magma version available
```

Comment:
is this an instruction to developer when changing the MAGMA_VERSION below?

Reply:
This is just a comment to trigger the workflows, I'll probably drop this commit once I verify that everything's run